spec/poros/quote_poro_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe QuotePoro do
```

Comment:
I would also test what happens when tags are empty since the logic for handling that is in the poro. If it were to break, we have no way of knowing that if it's not being tested.